app/controllers/stories_controller.rb

```diff
@@ -274,7 +274,7 @@ def assign_article_show_variables
   end
 
   def permission_denied?
-    !@article.published && params[:preview] != @article.password
+    (!@article.published || @article.scheduled?) && params[:preview] != @article.password
```

Comment:
Reading Notes: I really wish this could be part of the `ArticlePolicy` but that's a future concern.